app/views/candidate_interface/personal_details/immigration_right_to_work/_shared_form.html.erb

```diff
@@ -3,16 +3,29 @@
 </h1>
 
   <p class="govuk-body">
-    You’ll need the right to work or study in the UK for the duration of your course.
+    You’ll need to have the right to work or study for the duration of your course. You can use a visa which is due to expire before the end of your course, as long as you have the right to renew it.
   </p>
 
-  <p class="govuk-body"> <%= govuk_link_to('Check if you’ve already got the right to work or study in the UK, or whether you’ll need to apply for a visa',
-                    t('train_to_teach_in_england.url_citizen_visa')) %>.
+  <p class="govuk-body"> <%= govuk_link_to('Check your immigration status and find out if you need a visa',
+                    t('train_to_teach_in_england.url_citizen_visa'),
+                    target: :_blank,
+                    rel: 'noopener') %>.
+  </p>
+
+  <p class="govuk-body">
+    If you do not have the right to work or study, you can apply for a visa when you’ve been accepted onto a course that sponsors visas.
+  </p>
+
+  <p class="govuk-body">
+    Use the ’Visa sponsorship’ filter to <%= govuk_link_to('find courses that have visa sponsorship available',
```

Comment:
Sorry, I forgot to remove this line from the word doc (done now). I spoke with Laura and Maeve and we thought we should take this out as it's a bit lengthy and too many calls to action on the page taking them away from the service. Can we remove line 20 please?

Reply:
Done! Resolving now.